eoscostarica/guias.eoscostarica.io
# Docs(frontend):Update Guía Redes Sociales-186

### GH Issue

Resolves #186 

### Steps to test
1. Go to Guía para Redes Sociales.
2. Check content is right.


#### CheckList

- [ ] The content  is the translation of https://guide.eoscostarica.io/docs/social-media-guidelines
